parcel/ParcelModel.jl

```diff
@@ -112,16 +123,17 @@ function parcel_model(dY, Y, p, t)
 
     # number concentration and ...
     dNᵢ_dt = dNᵢ_dt_dep + dNᵢ_dt_imm + dNᵢ_dt_hom
-    dNₐ_dt = -dNᵢ_dt_dep
-    dNₗ_dt = -dNᵢ_dt_imm - dNᵢ_dt_hom
+    dNₐ_dt = -dNᵢ_dt_dep - dNₗ_dt_act
```

Comment:
It might become an issue that we are decreasing the number concentration of aerosol particles but not changing the standard deviation of the mode

Reply:
I'll make a note of that in documentation but maybe we can assume -- for the sake of simple parcel simulations for now -- that the aerosols are all activating with the same probability and that way the shape of the PDF doesn't change? Or is that a terrible assumption?